src/Sarif.Viewer.VisualStudio.UnitTests/CodeAnalysisResultManagerTests.cs

```diff
@@ -78,7 +78,7 @@ public void CodeAnalysisResultManager_GetRebaselinedFileName_AcceptsMatchingFile
                 this.commonDialogFactory);
 
             // Act.
-            string actualRebaselinedFileName = target.GetRebaselinedFileName(uriBaseId: null, fileName: FileNameInLogFile);
+            string actualRebaselinedFileName = target.GetRebaselinedFileName(uriBaseId: null, pathFromLogFile: FileNameInLogFile);
```

Comment:
Improved the parameter name in the product code. There are lots of "file names" and "paths" running around that code. It's important to use names that make it clear which is which.